apps/docs/src/app/platform/component-docs/platform-link/platform-link-examples/platform-link-standard-example.component.html

```diff
@@ -1,8 +1,8 @@
 <p>
     This is an Example of
-    <fdp-link [href]="'http://www.google.com'" [title]="'Extra info as tooltip text'">
-        <fd-icon [glyph]="'chain-link'"></fd-icon>
-        Standard Link
+    <fdp-link href="http://www.google.com" [title]="'Extra info as tooltip text'">
```

Comment:
@dimamarksman if we do not use brackets when using the title, the title is interpreted as a native HTML attribute, therefore it will be read twice, once for `fdp-link`, the second time for the actual HTML `a` inside the component. We have to use brackets when passing in the title because of that

Reply:
oh, this is not obvious thing